.changes/v2.24.0/661-improvements.md

```diff
@@ -0,0 +1 @@
+* `types.NsxtFirewallRule` adds field `ActionValue` instead of `Action` that is deprecated VCD API. It allows users to use `REJECT` option [GH-661]
```

Comment:
To match past changelog entries:
```suggestion
* Added field `ActionValue` to `types.NsxtFirewallRule` instead of `Action` that is deprecated in VCD API. It allows users to use `REJECT` option [GH-661]
```

Reply:
Fixed.